config/initializers/filter_parameter_logging.rb

```diff
@@ -2,3 +2,19 @@
 
 # Configure sensitive parameters which will be filtered from the log file.
 Rails.application.config.filter_parameters += [:password]
+
+# Personally identifiable information
+Rails.application.config.filter_parameters += [
+  :address_line_1,
+  :address_line_2,
+  :address_line_3,
+  :address_line_4,
+  :postcode,
+  :payroll_gender,
+  :teacher_reference_number,
+  :national_insurance_number,
+  :email_address,
+  :bank_sort_code,
+  :bank_account_number,
+  :SAMLResponse
+]
```

Comment:
Can we put these in the model? I suspect it will be easier to remember to keep them up to date if they're nearer to the code they refer to.

Also, if we want to test that we've made active decisions about all attributes, we could make it a hash, where the value determines whether or not we filter them, and we check that all the possible fields are a key in that hash.

For example on the `Claim` model:

```ruby
FILTER_PARAMETERS = {
  address_line_1: true,
  address_line_2: true,
  student_loan_plan: false,
}.freeze
```

Reply:
I really like this idea. We could have a spec that fails unless all attributes on the Claim model have been explicitly designated as filtered or not. So any future columns added without updating the hash would fail the test suite.  👍🏽